README.md

```diff
@@ -37,33 +42,53 @@ Integration expects the client to use REST to communicate and request actions su
 
 ```
 
+### Access Control Policy
+
+This is mostly defined by Hashicorp Vault. The application will use the `approle` authentication method to authenticate with the KMS. Meaning that different roles will have access to different keys.
+
+For most information, you can refer to the [Hashicorp Vault ACL documentation](https://www.vaultproject.io/docs/auth/approle).
+
+
 # Setup Development Environment
 
 ## Build and run docker
 ```bash
-docker compose up -d --build;
+docker compose up -d vault pawn;
 ```
```

Comment:
It would be good to add an addendum here (or in the `.env` section) that reminds users to redeploy the `pawn` service when the `.env` is updated.

Reply:
Will add

Reply:
updated :+1: